layers/meta-tegrademo/recipes-bsp/u-boot/u-boot-tegra_%.bbappend

```diff
@@ -0,0 +1,7 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI_append = " \
```

Comment:
Shouldn't this be a conditional append?  Otherwise, every machine will get this additional configuration.

Reply:
That's right.
I prefer this be a conditional append.